neps/utils/cli.py

```diff
@@ -579,6 +607,191 @@ def status(args: argparse.Namespace) -> None:
         print("-----------------------------")
 
 
+def results(args: argparse.Namespace) -> None:
+    """Handles the 'results' command by displaying incumbents, optionally plotting,
+    and dumping results to files based on the specified options."""
+    directory_path = get_root_directory(args)
+    if directory_path is None:
+        return
+
+    # Attempt to generate the summary CSV
+    try:
+        csv_config_data_path, _ = post_run_csv(directory_path)
+    except Exception as e:
+        print(f"Error generating summary CSV: {e}")
+        return
+
+    summary_csv_dir = csv_config_data_path.parent  # 'summary_csv' directory
+
+    # Load NePS state once
+    neps_state = load_neps_state(directory_path)
+    if neps_state is None:
+        return
+
+    # Load and sort all trials
+
+    trials = neps_state.get_all_trials()
+    sorted_trials = sorted(trials.values(), key=lambda x: int(x.id))
```

Comment:
This actually silently fails as a config id for multifidelity is actually something like `9_1`, which `int("9_1")` gets convereted to `91`.